hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeResourcePolicy.java

```diff
@@ -73,6 +79,11 @@ static boolean areResourcesAvailable(
       // required resources available.
       return requiredResourceCount > 0;
     } else {
+      if (minimumRedundantResources > resources.size()){
+        LOG.warn("The value of " + DFSConfigKeys.DFS_NAMENODE_CHECKED_VOLUMES_MINIMUM_KEY
```

Comment:
this is normal api call and false is genuine response. add a info log not warn. And just log resource not available, and put the values of all three variables. 

that is the most we can do, rest the admin need to be smart enough to decode what value is messed up and what damage it can cause

Reply:
I can completely understand your reply and have finished the relevant modification. Thank you very much for your suggestion:)